src/SFA.DAS.Data.Database/Views/DAS_Commitments.sql

```diff
@@ -112,11 +112,12 @@ SELECT [C].[ID]
                FROM
                     Data_Load.DAS_Employer_LegalEntities AS ELE
                WHERE
-                   IsLatest = 1
-               ) AS ELE ON C.LegalEntityOrganisationType = ELE.[LegalEntitySource]
-                          AND  C.[LegalEntityCode] = ELE.[LegalEntityNumber]
-                          AND C.[LegalEntityName] = ELE.LegalEntityName
+                   IsLatest = 1 AND
+				   C.LegalEntityOrganisationType = ELE.Source
```

Comment:
You need to do the ' ' replace here.  You can probably also reduce the number of fields returned by the outer apply as most aren't needed outside of the query now.